platform/ui-next/src/components/DataRow/DataRow.tsx

```diff
@@ -94,16 +96,19 @@ const DataRow: React.FC<DataRowProps> = ({
   isSelected = false,
   isVisible = true,
   disableEditing = false,
+  className,
 }) => {
   const [isDropdownOpen, setIsDropdownOpen] = useState(false);
   const isTitleLong = title?.length > 25;
   const rowRef = useRef<HTMLDivElement>(null);
 
-  useEffect(() => {
-    if (isSelected && rowRef.current) {
-      rowRef.current.scrollIntoView({ behavior: 'smooth', block: 'center' });
-    }
-  }, [isSelected]);
+  // useEffect(() => {
```

Comment:
@YiannisTheocharakis i'm removing this for now, it is buggy and has a lot of side effects, maybe we debug later